src/libraries/System.Formats.Tar/src/System/Formats/Tar/TarEntry.cs

```diff
@@ -98,7 +98,6 @@ public DateTimeOffset ModificationTime
             get => _header._mTime;
             set
             {
-                ArgumentOutOfRangeException.ThrowIfLessThan(value, DateTimeOffset.UnixEpoch);
```

Comment:
For `TarEntry.ModificationTime` and `PosixTarEntry.DeviceMinor` and `PosixTarEntry.DeviceMajor`:

I think we should still throw if the current entry's tar entry format is _not_ GNU and the value is beyond this limit for octal.

Reply:
> I think we should still throw if the current entry's tar entry format is not GNU and the value is beyond this limit for octal.

That makes sense. I checked and the GNU `tar` tool also doesn't want to use the GNU numeric format when using a non-GNU tar format. For reading it does accept the GNU numeric format independent of the archive tar format. I don't think we need to be as liberal. We don't expect to see the format in non-GNU archives, and it would require special handling in the tests because we wouldn't be able to use the writer to create the archives.